### ISSUE TITLE
Message received event in Voice Attack

### ISSUE THREAD

**Issue by Eclipse-73** [EDCD/EDDI#745]
## EDDI version in which issue found
EDDI version 3.0.0

## Steps to reproduce
1. Create a command in Voice Attack to be called by the Message received event '((EDDI message received))'
2. Trigger the event in-game, (receive a message)

## Expected
Voice attack command to be triggered once

## Observed
Voice Attack command was fired 4 times after the reception of a single message; consistently during my testing

## Investigation
I know this isn't the latest version and upon checking the change log, I was unable to find a mention of this issue being addressed
And I do not believe that there was anything else causing the script to fire repeatedly

**Comment by Tkael** [comment]
What is the content of the ((EDDI message received)) command you are using in VoiceAttack?

**Comment by Eclipse-73** [comment]
Since I'm unsure as to how I could do this in EDDI itself. (Which would alleviate my problem)
I copied this part of the command to Voice Attack so I could fiddle with it.  My goal was to have the name of the player only said if a new player messages me.
The EDDI message received script I've got has only the salute message part of it but other wise unchanged.
Attached is the command itself as well as a a screenshot of both it and the EDDI message received event script I'm using
![eddi message received script](https://user-images.githubusercontent.com/40684783/42973007-9a7b5b7c-8c05-11e8-916e-a17db0155db1.PNG)

![message received command](https://user-images.githubusercontent.com/40684783/42972678-55b53446-8c04-11e8-804e-3fad7bea8bfc.PNG)

[Message Received Command.zip](https://github.com/EDCD/EDDI/files/2211787/Message.Received.Command.zip)

**Comment by Eclipse-73** [comment]
Oh, and since I'm using an external file to set EDDI's Script variable for it to say. I've been check the output to that file during testing as well. This is working as intended on my end

**Comment by Tkael** [comment]
We're not observing this issue with the standard EDDI script and the default (blank) `((EDDI message received))` VoiceAttack command..
Does the VoiceAttack event vocalize / appear more than once when you use the default script and an empty `((EDDI message received))` command?

**Comment by Eclipse-73** [comment]
I've tried tested your suggestion and found that it worked as I would've expected.
I then tested multiple combinations of the EDDI event and the ((EDDI message received)) command... and despite my best efforts have now been unable to replicate my prior issue. Despite reverting back to exactly what I had before!

Everything seems to be working as intended at the moment, much to my confusion.
I will do some more testing and get back to you with the results of that, hopefully it'll work fine and all I'll have done is waste some your time. :[
